services/cache/lru.go

```diff
@@ -216,10 +217,9 @@ func (l *LRU) get(key string, value interface{}) error {
 			_, err := u.UnmarshalMsg(e.value)
 			*v = &u
 			return err
-		case **model.Session:
-			var s model.Session
-			_, err := s.UnmarshalMsg(e.value)
-			*v = &s
+		case *model.Session:
+			a := (interface{}(v)).(*model.Session)
+			_, err := a.UnmarshalMsg(e.value)
```

Comment:
This should not be needed. It is taken care of by L201-203.

Reply:
thanks and removed.